bwl_loot.json

```diff
@@ -356,8 +356,8 @@
   {
     "id": 19335,
     "name": "Spineshatter",
-    "role": "Tank-Warrior > Warrior",
-    "dkp": 120
+    "role": "Warrior",
+    "dkp": 60
```

Comment:
we ended up disenchanting this last raid so halfing DKP for the future. i believe the minbid was high on this because clouds initially wanted it for tanking